.github/workflows/crowdin-download.yaml

```diff
@@ -0,0 +1,35 @@
+name: Crowdin download
```

Comment:
**nitpick** probably can use `.yml` extension for these to be consistent, they do the same thing pretty sure.